File: TodoListUsingRoom/app/src/main/java/com/dodemy/android/todolistusingroom/Todo.java
package com.dodemy.android.todolistusingroom;

import androidx.room.Entity;
import androidx.room.Ignore;
import androidx.room.PrimaryKey;
import java.io.Serializable;

@Entity(tableName = MyDatabase.TABLE_NAME_TODO)
public class Todo implements Serializable {

    @PrimaryKey(autoGenerate = true)
    public int todo_id;

    public String name;

    public String description;

    public String category;

    @Ignore
    public String priority;

}

File: AndroidTesting/Kotlin/UnitTesting/TodoListWithRoomTesting/app/src/main/java/com/dodemy/todolistwithroomtesting/data/ToDo.kt
package com.dodemy.todolistwithroomtesting.data

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class ToDo(

    @PrimaryKey var id: String,
    var title: String,
    var dueDate: Long?,
    var completed: Boolean,
    var created: Long
)